requirements.in

```diff
@@ -29,8 +29,8 @@ wagtail==2.16.3
 wagtail-color-panel
 wagtail-factories
 wagtail-localize==1.1
-wagtail-localize-git==0.12.0
-wagtail-inventory
+wagtail-localize-git==0.13.0
+wagtail-inventory==1.6
```

Comment:
@kevinhowbrook Why are we pinning the version in the `requirements.in` file? Shouldn't this be resolved by `pip-compile` and locked in the `requirements.txt`?

Reply:
Thanks, localize-git was already pinned so I continued with that.
Have changed this though so both aren't pinned and are resolved with compile and lock

Reply:
Nice one. Thanks @kevinhowbrook.